intelligence-services/latest/using/index.md

```diff
@@ -45,7 +45,7 @@ Follow these steps to use the default (i.e. out-of-the-box) configuration of Int
         For example, you can add any of the default renditions:
 
         ```bash
-        aiFeatures, aiLabels, aiTextract
+        aiFeatures, aiLabels, aiTextract, aiSpeechToText, webvtt, aiPiiEntities 
         ```
 
         > **Note:** If you leave the text field empty, all three of the default renditions will be requested. The default renditions are `aiFeatures` for Amazon Comprehend, `aiLabels` for Amazon Rekognition of images, and `aiTextract` for Amazon Textract.
```

Comment:
@oliveralfresco2018 Also change the note. In fact, all six default renditions will now be requested (cc @pnirwal @pmartinezga).

You should probably also change theseTroubleshooting sections slightly to make them more generic or highlight that there are now six default renditions.

https://github.com/Alfresco/docs-alfresco/blob/master/intelligence-services/latest/admin/troubleshoot.md#what-happens-if-i-leave-the-request-ai-renditions-input-text-field-empty

What happens if I leave the 'Request AI renditions' input text field empty?
The default renditions are requested <add link to main doc section>.

https://github.com/Alfresco/docs-alfresco/blob/master/intelligence-services/latest/admin/troubleshoot.md#what-happens-if-i-specify-only-one-default-ai-rendition

`
What happens if I specify only one default AI rendition?
If you specify only one default AI rendition, such as aiFeatures, the other default renditions will be ignored. This may be really useful for saving costs if other renditions aren't required.
`

Reply:
@montgolfiere I shall leave a msg for you in the JIRA DOCS-6111

Reply:
I shall do these changes in sections because the PR was closed.
1. Corrected the Note:

Reply:
2. You should probably also change these related Troubleshooting sections slightly to make them more generic or highlight that there are now six default renditions.

Updated.

Reply:
Made all the changes on here. 
These changes will be visible in PR [DOCS-6111] Ux images added to ais 1.3 #220

Cheers @montgolfiere